setup-scripts/health-checker.py

```diff
@@ -211,59 +232,118 @@ async def check_health():
     force_notify = False
 
-    if len(critical_alerts) > 0:
-        message += "{} CRITICAL Alerts found! ".format(len(critical_alerts))
-        force_notify = True
-    if len(error_alerts) > 0:
-        message += "{} Error Alerts found! ".format(len(error_alerts))
+    if num_critical_alerts > 0:
+        message += "{} CRITICAL Alerts found! ".format(num_critical_alerts)
         force_notify = True
-    
-    message += "{} Warning Alerts found. ".format(len(warning_alerts))
-    message += "{} SiaFiles with bad health found. ".format(len(siafile_alerts))
+    if num_error_alerts > 0:
+        message += "{} Error Alerts found! ".format(num_error_alerts)
+        force_notify = num_error_alerts > num_siafile_alerts
 
-    alerts_file = None 
-    if len(alerts) > 0:
-        alerts_file = json.dumps(alerts, indent=2)
+    message += "{} Warning Alerts found. ".format(num_warning_alerts)
+    message += "{} SiaFiles with bad health found. ".format(num_siafile_alerts)
 
-    # send a message if we force notification, or just once daily (heartbeat) on 1 AM
+    # send a message if we force notification, or just once daily (heartbeat)
+    # on 1 AM
     if force_notify or datetime.utcnow().hour == 1:
         return await send_msg(
-            client, message, file=alerts_file, force_notify=force_notify
+            client, message, file=siac_alert_output, force_notify=force_notify
+        )
+
+
+# check_portal_size checks the number of files that the portal is managing to
+# determine if it is time to rotate it out
+async def check_portal_size():
+    print("\nChecking portal size...")
+
+    # Execute siac renter to check the size of the portal
+    #
+    # NOTE: we should leave this as always trying to execute the docker command
+    # against the sia container as this will then fail for maintenance severs
+    # were we don't care about this check.
+    cmd_string = "docker exec sia siac renter"
+    siac_renter_output = os.popen(cmd_string).read().strip()
+
+    # Initialize variables
+    num_files = 0
+    max_files = 250000
+    files_text = "Files"
+    for line in siac_renter_output.split("\n"):
+        if contains_string(line, files_text):
+            for el in line.split():
+                if el.isdigit():
+                    num_files = int(el)
+
+    ################################################################################
+    # create a message
+    ################################################################################
+
+    message = ""
+    force_notify = False
+
+    if num_files > max_files:
+        message += "Portal has {} files! Consider rotating! ".format(num_files)
+        # send notification when above 40% of the limit
+        force_notify = num_files > max_files * 1.4
+    else:
+        message += "Portal has {} files. ".format(num_files)
+
+    # send a message if we force notification, or just once daily (heartbeat) on 1 AM
+    if force_notify and datetime.utcnow().hour == 1:
```

Comment:
This `or` was intentional. For certain things we want to be pinged immediately (`force_notify`) and for others - once a day (do not force it). If we want to deviate from that logic let's change the name of the var, so it's clear what's happening.

Reply:
That was a mistake on my end. Didn't mean to change it to an `and`. Also forgot to add the Force Notify to the max files conditionally.